gdal/swig/python/scripts/gdal_retile.py

```diff
@@ -350,7 +350,8 @@ def tileImage(minfo, ti):
             if offsetY + height > ti.height:
                 height = ti.height - offsetY
 
-            createTile(minfo, offsetX, offsetY, width, height, tilename, OGRDS)
+            if not os.path.exists(tilename) or Overwrite:
```

Comment:
in default mode (skip if file exists), that assumes that an existing tile has been completely generated, which might not be true. So I think createTile() should be modified to write in a .tmp file, and once it is completed rename the .tmp file to its final name

Reply:
Hi, I'm also interested in this change. I would like to know how we should deal with it because AFAIU, the xml files produced are also important, so there are 2 files that need to be managed, and I don't know how to ensure atomicity.